config/phobos.yml.example

```diff
@@ -107,7 +107,11 @@ listeners:
     #  - `batch` will yield batches from Ruby Kafka using `each_batch`, and commit/heartbeat at every consumed batch.
     #     Due to implementation in Ruby Kafka, it should be noted that when configuring large batch sizes in combination with taking long time to consume messages,
     #     your consumers might get kicked from the Kafka cluster for not sending a heartbeat within the expected interval.
-    #     Take this into consideration when determining your configuration.
+    #     Take this into consideration when determining your configuration. You can send heartbeats manually while
+    #     processing your messages if necessary.
+    #   - `inline_batch` will use Phobos::BatchHandler instead of Phobos::Handler to pass the entire batch to your application
```

Comment:
I just realized when I wrote my latest comment that this is not really 100% accurate. It will pass to whatever consumer is configured so maybe we should phrase this something like

```
- `inline_batch` will pass the entire batch to your consumer, so make sure that this consumer includes `Phobos::BatchHandler` instead of `Phobos::Handler`
```

Reply:
Yep, will do!